msg/OrientationConstraint.msg

```diff
@@ -8,10 +8,20 @@ geometry_msgs/Quaternion orientation
 # The robot link this constraint refers to
 string link_name
 
-# optional axis-angle error tolerances specified
+# Optional symmetric error tolerances specified
 float64 absolute_x_axis_tolerance
 float64 absolute_y_axis_tolerance
 float64 absolute_z_axis_tolerance
 
+# How should we parameterize the optional orientation error tolerance?
+# See absolute axis tolerance below to specify the manitude.
```

Comment:
```suggestion
# See absolute axis tolerance below to specify the magnitude.
```
1. The tolerances are specified above this comment. I think it should be above the tolerances though, as you write here.
2. VSCode has a spell checker, once again~